docs/api/analyzer.md

```diff
@@ -0,0 +1,81 @@
+# Presidio Analyzer API Reference
```

Comment:
Specifying explicitly as there are many classes which aren't important and would make this extremely long. WDYT?

Reply:
wouldnt it be covered by the swagger spec?if it spepcifcly the python package API maybe rename the file to python_api.md?